posts/2023-06-21-data-protection-compliance-with-distributed-ledger-erasure.md

```diff
@@ -0,0 +1,241 @@
+                               }
+                            }
+                        },
+                        "x-vendia-erasable": [
+                            "number", 
+                            "aliases"
+                        ]
+                    },
+                    "firstName": {
+                        "type": "string"
+                    },
+                    "lastName": {
+                        "type": "string"
+                    }
+                },
+                "x-vendia-erasable": [
+                    "firstName",
+                    "lastName"
+                ]
+            }
+        }
+    }
+}
+```
+
+Once declared, Vendia will generate `erase` mutations for all data types with erasable fields.
+
+Field-level erasure can then be performed using a simple GraphQL mutation:
+
+```graphql
+mutation erase {
+  erase_Person(
+    id: "0187bd97-0593-cb3e-9c76-b71438598915"
+    input: {firstName: GLOBAL, lastName: GLOBAL, identification: {number: GLOBAL} }
+  ) {
+    result {
+      _id
+    }
+  }
+}
+```
+
+File data can be erased using `eraseVendia_File` mutation which will permanently erase **all** versions of the File from the CSP object store. i.e.
+```graphql
+mutation erase {
+  eraseVendia_File(
+    id: "0187bd97-0593-cb3e-9c76-b71438598915"
+    eraseMode: LOCAL
+    syncMode: NODE_COMMITTED
+  ) {
+    result {
+      _id
+    }
+  }
+}
+```
+
+Note that Vendia supports both `LOCAL` (current node) and `GLOBAL` (all nodes in Uni) [erasure modes](https://www.vendia.com/docs/share/erasure#local-vs-global-erasure).
+
+[Permissions to erase](https://www.vendia.com/docs/share/erasure#managing-erasure-permissions) can be managed on a per-object basis.
+
+## Auditability
+
+Auditability is another important requirement for compliance regimes such as GDPR and CCPA. Vendia stores an immutable record
+of start and completion for all erasure operations directly in the ledger.
+
+i.e.
+```graphql
+mutation m {
+    updateVendia_EraseTask(
+        id: "0187c2e5-2e10-341c-12fe-5acb9251bb6d", 
+        input: {
+            completionTime: \"2023-04-27T13:26:53.645450791+00:00\", 
+            status: SUCCESS
+        }) {error}
+}
+```
+
+## Masking
+
+The Vendia Share ledger also supports Masking, which ensures that previously erased data can never be set 
+again by further mutations to the object. The field
+will be masked in subsequent mutations and treated as redacted. For globally erased fields it will be masked on all nodes.
+For locally erased fields, it wil be masked on the nodes on which it has been locally erased.
+
+
+## Conclusion
+
+Data erasure capabilities should be a key consideration for data controllers managing personal information and requiring compliance with data protection regimes such as GDPR and CCPA. 
+Vendia Share provides a fully compliant and easy to use data sharing solution that 
+also provides the important properties of distributed ledgers.
+
```

Comment:
I would recommend to add a call to action here. Such as for additional information, where to look. Or a link to our registration page so they can register and play around with this new feature.